packages/ai-jsx/src/batteries/sidekick/platform/system-message.tsx

```diff
@@ -52,10 +51,7 @@ export function SidekickSystemMessage({
     <>
       {dateTimeSystemMessage}
       {responseFramingSystemMessage}
-      <SystemMessage>
-        <Prompt apolitical concise persona={`expert ${role}`} />
-        {responseFramingSystemMessage}
-      </SystemMessage>
+      <SystemMessage>{responseFramingSystemMessage}</SystemMessage>
```

Comment:
Lines 53 and 54 are redundant

Reply:
jfc lol. hey @zkoch in case you were wondering why our system message feels bloated, it's in part because we're literally putting the same content in multiple times.